Karma/Models/Charity.cs

```diff
@@ -69,5 +91,18 @@ public List<ItemType> GetItemTypes()
             return itemTypes.Any() ? itemTypes : null;
         }
 
+        protected virtual void OnCharityStateChanged(CharityStateChangedEventArgs e)
+        {
+            EventHandler<CharityStateChangedEventArgs> handler = CharityStateChanged;
+            handler?.Invoke(this, e);
+        }
```

Comment:
Are we planning to extend Charity class and override this method? If not, `virtual` is unnecessary.

Reply:
Yeah, that's true. Fixed